less/navigation.less

```diff
@@ -1 +1,49 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+nav {
+    text-align: center;
+    padding: 5px 5px 0px 5px;
+    
+
+  a{
+    
+    padding: 5px 5px 0px 5px;
```

Comment:
I see you have applied the same padding on both the `a` and the `nav` tags. I am assuming you try to get a behavior that didn't work and moved on forgetting to clean up one of them. You don't want to forget to clean up as this can cause some funky behaviors you didn't intend once you finally have the original bug fixed.